tests/Features/Models/Relations.php

```diff
@@ -141,6 +141,14 @@ public function testRelationWithTrait(Account $account): BelongsTo
     {
         return $account->ownerRelation();
     }
+
+    /** @test */
+    public function it_has_relationship_count_properties(): void
+    {
+        User::withCount('accounts')->get()->each(function (User $user) {
+            echo $user->accounts_count;
```

Comment:
Instead of echoing it here, can you return it and check the return type is an integer?